docs/sts.md

```diff
@@ -176,7 +176,7 @@ $ ./ccoctl aws create-iam-roles --output-dir ./newstscluster/ --name newstsclust
 
 13. Apply the Secrets generated by the above command:
 ```bash
-$ find ./cred-reqs -iname "*yaml" -print0 | xargs -I {} -0 -t oc replace -f {}
+$ find ./newstscluster -iname "*yaml" -print0 | xargs -I {} -0 -t oc replace -f {}
```

Comment:
This can be more precise by using `find ./newstscluster/manifests`.

Reply:
updated as requested